test/test_brt.py

```diff
@@ -40,13 +40,78 @@ def test_brt_p(mock_nfs_mount):
 
 @pytest.mark.localdata
 @pytest.mark.slow
-def test_brt_callback(mock_nfs_mount, caplog, mock_callback_data):
+def test_brt_server_response(mock_nfs_mount, caplog, mock_callback_data):
     from em_workflows.brt.flow import brt_flow
 
     input_dir = "test/input_files/brt/Projects/RT_TOMO/"
     if not Path(input_dir).exists():
         pytest.skip(f"Directory {input_dir} doesn't exist")
 
+    state = brt_flow(
```

Comment:
add note stating these params are EXTREMELY sensitive. Modding anything will likely result in BRT not producing required outputs.

Reply:
I believe, tests should be updated if necessary.

Reply:
Sorry - I'm not following

Reply:
Nevermind, adding it to the test comments.